docker/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ x-logging: &logging
 
 services:
   magi:
-    image: a16zcrypto/magi:latest
+    # image: a16zcrypto/magi:latest
```

Comment:
Why is this commented out?

Reply:
I commented it out to let the image be built from the Dockerfile locally for testing purposes, as I can't push new tags to the a16zcrypto org on dockerhub

Reply:
Ah we actually have the run-local make command for thi